### PR TITLE
feat: USWDS 2.13.3

### LINKED ISSUE #2174: [feat] Upgrade to USWDS 2.13.3
**Does your feature request relate to a specific USWDS component?**

**What USWDS Version is this feature present in?**

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by suzubara**
# Summary

This PR upgrades USWDS to 2.13.3, and makes minimal changes required to support that version.
* Update to small Search markup (non-breaking since markup is implemented by this lib)
* Update to Footer social button markup (these icons are implemented as children so users may need to update their implementation)
* Adds missing `aria-controls` prop to ComboBox input
* Use `aria-describedby` for DatePicker hint, and `aria-labelledby` for DatePicker label (instead of combining into a single attribute)

### Upgrades include:
* 2.13.0 [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.0) and [diff](https://github.com/uswds/uswds/compare/v2.12.2...v2.13.0)
* 2.13.1 [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.1) and [diff](https://github.com/uswds/uswds/compare/v2.13.0...v2.13.1)
* 2.13.2 [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.2) and [diff](https://github.com/uswds/uswds/compare/v2.13.1...v2.13.2)
* 2.13.3 [release notes](https://github.com/uswds/uswds/releases/tag/v2.13.3) and [diff](https://github.com/uswds/uswds/compare/v2.13.2...v2.13.3)

### Other changes:
* **New component:** `SocialLink` (added for convenience when using `<SocialLinks>` in the Footer)

closes #2174

### CODE REVIEW

File: src/components/forms/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -401,6 +401,7 @@ const ComboBoxForwardRef: React.ForwardRefRenderFunction<
         value={state.inputValue}
         focused={state.focusMode === FocusMode.Input}
         aria-owns={listID}
+        aria-controls={listID}
```
Comment (suzubara) [inline src/components/forms/ComboBox/ComboBox.tsx:404]: Ref: https://github.com/uswds/uswds/compare/v2.13.1...v2.13.2#diff-b5b7f2af368629dba201dda55dd013693cfcdfdb98dcf2ff16d7a079723b63ffR213

File: src/components/forms/DatePicker/Calendar.tsx [hunk 2]
```diff
@@ -363,9 +363,7 @@ export const Calendar = ({
             ref={prevYearEl}
             className="usa-date-picker__calendar__previous-year"
             aria-label={backOneYear}
-            disabled={prevButtonsDisabled}>
-            &nbsp;
-          </button>
+            disabled={prevButtonsDisabled}></button>
```
Comment (suzubara) [inline src/components/forms/DatePicker/Calendar.tsx:366]: ref: https://github.com/uswds/uswds/compare/v2.13.2...v2.13.3#diff-ffadd89070c3575a2961e89afb4498a888ae01c38890d42c5d760ce82ea6d487

File: src/components/forms/DatePicker/DatePicker.stories.tsx [hunk 3]
```diff
@@ -61,7 +61,8 @@ export const completeDatePicker = (
       <DatePicker
         id="appointment-date"
         name="appointment-date"
-        aria-describedby="appointment-date-label appointment-date-hint"
+        aria-describedby="appointment-date-hint"
+        aria-labelledby="appointment-date-label"
```
Comment (suzubara) [inline src/components/forms/DatePicker/DatePicker.stories.tsx:65]: ref: https://github.com/uswds/uswds/compare/v2.12.2...v2.13.0#diff-85ddb26937b3016aaddc3058619885bb9d66a2c0439d2162b262ab1f236e1349R17

File: src/components/forms/DateRangePicker/DateRangePicker.tsx [hunk 4]
```diff
@@ -181,7 +171,8 @@ export const DateRangePicker = (
           className={endDatePickerClasses}
           rangeDate={startDateInternalValue}
           {...endDatePickerProps}
-          aria-describedby={endDatePickerAriaDescribedBy}
+          aria-labelledby={endDateLabel && endDatePickerLabelId}
+          aria-describedby={endDateHint && endDatePickerHintId}
```
Comment (suzubara) [inline src/components/forms/DateRangePicker/DateRangePicker.tsx:175]: ref: https://github.com/uswds/uswds/compare/v2.12.2...v2.13.0#diff-adde064603648d5f09dbae989bacac0a8b336e100ea9520b67e7a2887debd75d

File: src/components/forms/DatePicker/DatePicker.stories.tsx (hunk 3 above)
Comment (brandonlenz) [reply src/components/forms/DatePicker/DatePicker.stories.tsx:65]: The links aren't working for me, but I was able to find and confirm what I think you're pointing to: 
image

File: src/components/forms/DateRangePicker/DateRangePicker.tsx (hunk 4 above)
Comment (brandonlenz) [reply src/components/forms/DateRangePicker/DateRangePicker.tsx:175]: image

File: src/stories/templates/createaccount.stories.tsx [hunk 5]
```diff
@@ -52,6 +53,14 @@ const returnToTop = (
   </GridContainer>
 )
 
+const socialLinkItems = [
+  <SocialLink key="facebook" name="Facebook" href="#" />,
```
Comment (brandonlenz) [inline src/stories/templates/createaccount.stories.tsx:57]: Nice. This feels more in line with the way we're composing newer components.

**Review by brandonlenz — APPROVED** [review]

**Pull request merged**